### PR TITLE
Fixed issue with scrolling to elements; Improved logging; Repository helpers, views and widgets

### PR DESCRIPTION
**Pull request by abalakh**
Summary of changes:
* Fixed issue with taxonomies list hovering page when scrolling bottom-up (see screenshot)
* Removed flooding `selenium.webdriver.remote.remote_connection` messages from debug level which are redundant in 99% cases and were only useful on the earliest stages of development
* Fixed issue with `EditableEntry` widget when it wasn't able to fetch value of entries with similar names (e.g. 'publish via http' because of 'publish via https' present) due to `contains()` nature
* Added helpers, views and widgets for Repository's Packages and Puppet Modules

image

### CODE REVIEW

File: airgun/entities/repository.py [hunk 1]
```diff
@@ -49,6 +53,46 @@ def synchronize(self, product_name, entity_name):
...
+            view.select_all.fill(True)
+            view.remove_packages.click()
+            view.dialog.confirm()
+            view.flash.assert_no_error()
+            view.flash.dismiss()
+        if view.total_packages.text != '0':
+            raise AssertionError(
+                'Unable to remove all packages from {}'.format(entity_name))
+
+    def remove_puppet_modules(self, product_name, entity_name):
```
Comment (oshtaier) [inline airgun/entities/repository.py]: same here

File: airgun/entities/repository.py [hunk 2]
```diff
@@ -49,6 +53,46 @@ def synchronize(self, product_name, entity_name):
...
 
+    def remove_packages(self, product_name, entity_name):
+        """Remove all packages from repository"""
+        view = self.navigate_to(
+            self,
+            'Packages',
+            product_name=product_name,
+            entity_name=entity_name,
+        )
+        view.items_per_page.fill('100')
```
Comment (oshtaier) [inline airgun/entities/repository.py]: maybe we can select latest value in the list, not the static `100`? or maybe no =)

File: airgun/entities/repository.py [hunk 3]
```diff
@@ -49,6 +53,46 @@ def synchronize(self, product_name, entity_name):
         view.progressbar.wait_for_result()
         return view.read()
 
+    def remove_packages(self, product_name, entity_name):
```
[the remark this answers is not in the document]
Comment (abalakh) [reply airgun/entities/repository.py]: To think of it, to remove specific packages we'll use searchbar and won't use pagination or conditional loops at all - basically, that'll be entirely new method with almost nothing in common, so i'd rather rename it to `remove_all_packages` instead and implement `remove_package` when needed.

File: airgun/entities/repository.py (hunk 2 above)
Comment (abalakh) [reply airgun/entities/repository.py]: latest is probably not the safest idea. I can see another option - fetch all the values, convert to int, find max, select it (and use that value in range of `for` loop). But that may be a bit overcomplicated. Thoughts?

Comment (oshtaier) [reply airgun/entities/repository.py]: yeah, that was my initial idea, but probably it doesn't worth to spend efforts on that

**Comment by abalakh** [comment]
@oshtaier 
* comments addressed
* test results

<details>
 <summary>Test Results</summary>

```
pytest -v tests/foreman/ui_airgun/test_activationkey.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 25 items
2018-06-19 14:15:43 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_activationkey.py::test_positive_create PASSED [  4%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_delete PASSED [  8%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_edit PASSED [ 12%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_create_with_cv[0] PASSED [ 16%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_search_scoped PASSED [ 20%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_create_with_host_collection PASSED [ 24%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_create_with_envs PASSED [ 28%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_add_host_collection_non_admin PASSED [ 32%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_remove_host_collection_non_admin PASSED [ 36%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_delete_with_env PASSED [ 40%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_delete_with_cv PASSED [ 44%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_update_env PASSED [ 48%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_update_cv[0] FAILED [ 52%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_update_rh_product FAILED [ 56%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_add_rh_product PASSED [ 60%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_add_custom_product PASSED [ 64%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_add_rh_and_custom_products PASSED [ 68%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_fetch_product_content PASSED [ 72%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_access_non_admin_user PASSED [ 76%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_remove_user PASSED [ 80%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_add_host PASSED [ 84%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_delete_with_system PASSED [ 88%]
tests/foreman/ui_airgun/test_activationkey.py::test_negative_usage_limit PASSED [ 92%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_add_multiple_aks_to_system PASSED [ 96%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_host_associations FAILED [100%]

==================== 3 failed, 21 passed in 4053.80 seconds ====================

pytest -v tests/foreman/ui_airgun/test_architecture.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 3 items
2018-06-19 16:27:30 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_architecture.py::test_positive_create[0] PASSED [ 33%]
tests/foreman/ui_airgun/test_architecture.py::test_positive_create_with_os PASSED [ 66%]
tests/foreman/ui_airgun/test_architecture.py::test_positive_delete[0] PASSED [100%]

========================== 3 passed in 102.66 seconds ==========================

pytest -v tests/foreman/ui_airgun/test_computeprofiles.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 3 items
2018-06-19 16:32:35 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_computeprofiles.py::test_positive_create PASSED [ 33%]
tests/foreman/ui_airgun/test_computeprofiles.py::test_positive_rename PASSED [ 66%]
tests/foreman/ui_airgun/test_computeprofiles.py::test_positive_delete PASSED [100%]

========================== 3 passed in 100.51 seconds ==========================

pytest -v tests/foreman/ui_airgun/test_contentview.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 20 items
2018-06-19 15:22:37 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_contentview.py::test_positive_create PASSED [  5%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_custom_content PASSED [ 10%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_end_to_end PASSED [ 15%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_repo_count_for_composite_cv PASSED [ 20%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_puppet_module PASSED [ 25%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_create_composite PASSED [ 30%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_rh_content PASSED [ 35%]
tests/foreman/ui_airgun/test_contentview.py::test_negative_add_puppet_repo_to_composite PASSED [ 40%]
tests/foreman/ui_airgun/test_contentview.py::test_negative_add_components_to_non_composite PASSED [ 45%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_unpublished_cv_to_composite PASSED [ 50%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_non_composite_cv_to_composite PASSED [ 55%]
tests/foreman/ui_airgun/test_contentview.py::test_negative_add_dupe_repos PASSED [ 60%]
tests/foreman/ui_airgun/test_contentview.py::test_negative_add_dupe_modules PASSED [ 65%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_publish_with_custom_content PASSED [ 70%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_publish_with_rh_content PASSED [ 75%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_publish_composite_with_custom_content PASSED [ 80%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_publish_version_changes_in_target_env PASSED [ 85%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_promote_with_custom_content PASSED [ 90%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_promote_with_rh_content PASSED [ 95%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_promote_composite_with_custom_content PASSED [100%]

========================= 20 passed in 3709.42 seconds =========================


pytest -v tests/foreman/ui_airgun/test_contentcredential.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 17 items
2018-06-19 16:39:32 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_create_via_import PASSED [  5%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_add_empty_product PASSED [ 11%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_add_product_with_repo PASSED [ 17%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_add_product_with_repos PASSED [ 23%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_add_repo_from_product_with_repo PASSED [ 29%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_add_repo_from_product_with_repos PASSED [ 35%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_add_product_using_repo_discovery FAILED [ 41%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_update_key_for_empty_product PASSED [ 47%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_update_key_for_product_with_repo PASSED [ 52%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_update_key_for_product_with_repos PASSED [ 58%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_update_key_for_repo_from_product_with_repo PASSED [ 64%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_update_key_for_repo_from_product_with_repos PASSED [ 70%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_delete_key_for_empty_product PASSED [ 76%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_delete_key_for_product_with_repo PASSED [ 82%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_delete_key_for_product_with_repos PASSED [ 88%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_delete_key_for_repo_from_product_with_repo PASSED [ 94%]
tests/foreman/ui_airgun/test_contentcredential.py::TestGPGKeyProductAssociate::test_positive_delete_key_for_repo_from_product_with_repos PASSED [100%]

==================== 1 failed, 16 passed in 1139.89 seconds ====================

pytest -v tests/foreman/ui_airgun/test_domain.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 9 items
2018-06-19 18:51:47 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_domain.py::test_positive_create_with_name[0] PASSED [ 11%]
tests/foreman/ui_airgun/test_domain.py::test_positive_delete PASSED      [ 22%]
tests/foreman/ui_airgun/test_domain.py::test_positive_update[0] PASSED   [ 33%]
tests/foreman/ui_airgun/test_domain.py::test_negative_create_with_invalid_name[0] PASSED [ 44%]
tests/foreman/ui_airgun/test_domain.py::test_positive_set_parameter[long_value] PASSED [ 55%]
tests/foreman/ui_airgun/test_domain.py::test_positive_set_parameter[blank_value] PASSED [ 66%]
tests/foreman/ui_airgun/test_domain.py::test_negative_set_parameter PASSED [ 77%]
tests/foreman/ui_airgun/test_domain.py::test_negative_set_parameter_same PASSED [ 88%]
tests/foreman/ui_airgun/test_domain.py::test_positive_remove_parameter PASSED [100%]

========================== 9 passed in 444.77 seconds ==========================

pytest -v tests/foreman/ui_airgun/test_hostcollection.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 2 items
2018-06-19 19:00:59 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_hostcollection.py::test_positive_create PASSED [ 50%]
tests/foreman/ui_airgun/test_hostcollection.py::test_positive_add_host PASSED [100%]

========================== 2 passed in 149.77 seconds ==========================


pytest -v tests/foreman/ui_airgun/test_lifecycleenvironment.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 2 items
2018-06-19 19:15:35 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_lifecycleenvironment.py::test_positive_edit PASSED [ 50%]
tests/foreman/ui_airgun/test_lifecycleenvironment.py::test_positive_create_chain PASSED [100%]

========================== 2 passed in 83.08 seconds ===========================

pytest -v tests/foreman/ui_airgun/test_location.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 1 item
2018-06-19 19:38:55 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_location.py::test_positive_create PASSED    [100%]

========================== 1 passed in 38.60 seconds ===========================


pytest -v tests/foreman/ui_airgun/test_operatingsystem.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 7 items
2018-06-19 19:46:26 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_operatingsystem.py::test_positive_create[0] PASSED [ 14%]
tests/foreman/ui_airgun/test_operatingsystem.py::test_positive_create_with_arch PASSED [ 28%]
tests/foreman/ui_airgun/test_operatingsystem.py::test_positive_create_with_ptable PASSED [ 42%]
tests/foreman/ui_airgun/test_operatingsystem.py::test_positive_create_with_ptable_same_org PASSED [ 57%]
tests/foreman/ui_airgun/test_operatingsystem.py::test_positive_create_with_params PASSED [ 71%]
tests/foreman/ui_airgun/test_operatingsystem.py::test_positive_update_with_params PASSED [ 85%]
tests/foreman/ui_airgun/test_operatingsystem.py::test_positive_delete PASSED [100%]

========================== 7 passed in 316.03 seconds ==========================

pytest -v tests/foreman/ui_airgun/test_organization.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 11 items
2018-06-19 20:08:37 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_organization.py::test_positive_create PASSED [  9%]
tests/foreman/ui_airgun/test_organization.py::test_positive_update_user PASSED [ 18%]
tests/foreman/ui_airgun/test_organization.py::test_positive_create_with_all_users SKIPPED [ 27%]
tests/foreman/ui_airgun/test_organization.py::test_positive_update_compresource PASSED [ 36%]
tests/foreman/ui_airgun/test_organization.py::test_positive_update_media PASSED [ 45%]
tests/foreman/ui_airgun/test_organization.py::test_positive_update_template PASSED [ 54%]
tests/foreman/ui_airgun/test_organization.py::test_positive_update_ptable PASSED [ 63%]
tests/foreman/ui_airgun/test_organization.py::test_positive_update_domain PASSED [ 72%]
tests/foreman/ui_airgun/test_organization.py::test_positive_update_environment PASSED [ 81%]
tests/foreman/ui_airgun/test_organization.py::test_positive_update_hostgroup PASSED [ 90%]
tests/foreman/ui_airgun/test_organization.py::test_positive_update_location PASSED [100%]

==================== 10 passed, 1 skipped in 915.16 seconds ====================


pytest -v tests/foreman/ui_airgun/test_partitiontable.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 8 items
2018-06-19 20:30:51 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_partitiontable.py::test_positive_create_default_for_organization PASSED [ 12%]
tests/foreman/ui_airgun/test_partitiontable.py::test_positive_create_custom_organization PASSED [ 25%]
tests/foreman/ui_airgun/test_partitiontable.py::test_positive_create_default_for_location PASSED [ 37%]
tests/foreman/ui_airgun/test_partitiontable.py::test_positive_create_custom_location PASSED [ 50%]
tests/foreman/ui_airgun/test_partitiontable.py::test_positive_delete PASSED [ 62%]
tests/foreman/ui_airgun/test_partitiontable.py::test_positive_update PASSED [ 75%]
tests/foreman/ui_airgun/test_partitiontable.py::test_positive_delete_with_lock_and_unlock PASSED [ 87%]
tests/foreman/ui_airgun/test_partitiontable.py::test_positive_clone PASSED [100%]

========================== 8 passed in 363.45 seconds ==========================


pytest -v tests/foreman/ui_airgun/test_product.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 1 item
2018-06-19 20:43:41 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_product.py::test_positive_create_in_different_orgs[0] PASSED [100%]

========================== 1 passed in 101.37 seconds ==========================


pytest -v tests/foreman/ui_airgun/test_provisioningtemplate.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 2 items
2018-06-19 20:45:39 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_provisioningtemplate.py::test_positive_create PASSED [ 50%]
tests/foreman/ui_airgun/test_provisioningtemplate.py::test_positive_clone PASSED [100%]

========================== 2 passed in 80.67 seconds ===========================

pytest -v tests/foreman/ui_airgun/test_repository.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 12 items
2018-06-19 20:56:47 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_repository.py::test_positive_create_in_different_orgs PASSED [  8%]
tests/foreman/ui_airgun/test_repository.py::test_positive_create_as_non_admin_user PASSED [ 16%]
tests/foreman/ui_airgun/test_repository.py::test_positive_create_puppet_repo_same_url_different_orgs PASSED [ 25%]
tests/foreman/ui_airgun/test_repository.py::test_positive_create_as_non_admin_user_with_cv_published PASSED [ 33%]
tests/foreman/ui_airgun/test_repository.py::test_positive_discover_repo_via_existing_product PASSED [ 41%]
tests/foreman/ui_airgun/test_repository.py::test_positive_discover_repo_via_new_product PASSED [ 50%]
tests/foreman/ui_airgun/test_repository.py::test_positive_sync_custom_repo_yum PASSED [ 58%]
tests/foreman/ui_airgun/test_repository.py::test_positive_sync_custom_repo_puppet PASSED [ 66%]
tests/foreman/ui_airgun/test_repository.py::test_positive_sync_custom_repo_docker PASSED [ 75%]
tests/foreman/ui_airgun/test_repository.py::test_positive_resync_custom_repo_after_invalid_update PASSED [ 83%]
tests/foreman/ui_airgun/test_repository.py::test_positive_resynchronize_rpm_repo PASSED [ 91%]
tests/foreman/ui_airgun/test_repository.py::test_positive_resynchronize_puppet_repo PASSED [100%]

========================= 12 passed in 1533.02 seconds =========================

pytest -v tests/foreman/ui_airgun/test_role.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 5 items
2018-06-19 21:46:19 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_role.py::test_positive_update_filter PASSED [ 20%]
tests/foreman/ui_airgun/test_role.py::test_positive_delete_filter PASSED [ 40%]
tests/foreman/ui_airgun/test_role.py::test_positive_create_filter_without_override PASSED [ 60%]
tests/foreman/ui_airgun/test_role.py::test_positive_create_non_overridable_filter PASSED [ 80%]
tests/foreman/ui_airgun/test_role.py::test_positive_create_overridable_filter PASSED [100%]

========================== 5 passed in 466.32 seconds ==========================

pytest -v tests/foreman/ui_airgun/test_subnet.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 2 items
2018-06-19 22:08:43 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_subnet.py::test_positive_create PASSED      [ 50%]
tests/foreman/ui_airgun/test_subnet.py::test_positive_create_v6 PASSED   [100%]

========================== 2 passed in 71.54 seconds ===========================

pytest -v tests/foreman/ui_airgun/test_syncplan.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 3 items
2018-06-19 22:36:41 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_syncplan.py::test_positive_create PASSED    [ 33%]
tests/foreman/ui_airgun/test_syncplan.py::test_positive_create_with_start_time PASSED [ 66%]
tests/foreman/ui_airgun/test_syncplan.py::test_positive_create_with_start_date FAILED [100%]

===================== 1 failed, 2 passed in 173.22 seconds =====================

pytest -v tests/foreman/ui_airgun/test_user.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 6 items
2018-06-19 22:40:18 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_user.py::test_positive_create_with_multiple_roles PASSED [ 16%]
tests/foreman/ui_airgun/test_user.py::test_positive_create_with_all_roles PASSED [ 33%]
tests/foreman/ui_airgun/test_user.py::test_positive_create_with_multiple_orgs PASSED [ 50%]
tests/foreman/ui_airgun/test_user.py::test_positive_update_with_multiple_roles PASSED [ 66%]
tests/foreman/ui_airgun/test_user.py::test_positive_update_with_all_roles PASSED [ 83%]
tests/foreman/ui_airgun/test_user.py::test_positive_update_orgs PASSED   [100%]

========================== 6 passed in 489.37 seconds ==========================
```
</details>

* Console output for repo test http://pastebin.test.redhat.com/605675

**Pull request merged**